dlhub_sdk/models/servables/python.py

```diff
@@ -153,8 +155,8 @@ class PythonStaticMethodModel(BasePythonServableModel):
     """
 
     @classmethod
-    def create_model(cls, module, method, autobatch=False, function_kwargs=None):
-        """Initialize the method
+    def create_model(cls, module=None, method=None, autobatch=False, function_kwargs=None, *, f=None):
```

Comment:
A little hard to explain in a comment here, but it's an issue of backwards compatibility for the `create_model` function. `f` is an entirely new parameter in this version of the function, so to avoid breaking old code the positions of the existing parameters need to remain the same. It would look weird then to use positionally, because you would have to specify `None` for the parameters you wanted to ignore. I hope this made sense.